vic5_manuscript.tex

```diff
@@ -220,7 +224,7 @@ \section{VIC Infrastructure}
     VIC previously included a meteorological forcing processor that made use of the MT-CLIM algorithms to estimate radiation and humidity from daily inputs of precipitation and temperature and which created sub-daily forcings from daily inputs. In VIC-5, this forcing processor has been removed from the model to facilitate a number of key enhancements that are described below:
 
     \begin{enumerate}
-      \item \textit{Exact restarts}: In the versions of VIC that included MT-CLIM, exact restarts were not possible due to a rolling time average in MT-CLIM. In practice, applications that required exact restarts for short model runs (e.g. hydrologic forecasting) were forced to run the model for long spin-up periods to achieve near-exact restarts.
+      \item \textit{Exact restarts}: In the versions of VIC that included MT-CLIM, exact restarts were not possible due to a rolling time average in MT-CLIM. By exact restarts, we mean that the model was not able to save and restore its state without loosing information. In practice, applications that required the ability to continue previous simulations using the ending model state (e.g. hydrologic forecasting) were forced to run the model for long spin-up periods to achieve satisfactory restart performance (not exact).

```

Comment:
1. should be "losing", not "loosing"
2. how about "In the versions of VIC that included MT-CLIM, exact restarts from a previously-saved state were not possible, due to MT-CLIM's dependence on a rolling 90-day window of previous meteorological forcings.  Because these forcings were not included in the VIC state file, MT-CLIM used a default initial condition during restarts, causing a discontinuity in model state. In practice, applications that required the ability to continue previous simulations using the ending model state (e.g. hydrologic forecasting) were forced to run the model for long spin-up periods to achieve satisfactory (but inexact) restart performance."  I realize this is longer, probably too wordy, but maybe clearer?

Reply:
in Ted's suggested edit: Way too much detail about MT-CLIM for my taste.

In the original edit, we want to rephrase this. "By exact restarts, we mean that ..." This is a bit clunky, because then it ends up describing what an exact restart is not rather than what it is.

Reply:
How about something like: "In the versions of VIC that included MT-CLIM, exact restarts were not possible due to a rolling time average. By removing MT-CLIM in VIC.5, it is now possible to end a run and save the model state at that timestep and then start a new model run with exactly the same model state. This improvement has widespread applicability in hydrologic modeling and is particularly important for forecasting applications."